handlers/nonini_wpt_meta/__init__.py

```diff
@@ -0,0 +1,34 @@
+from eventhandler import EventHandler
+
+NON_INI_MSG = 'This pull request adds {0} without the .ini \
+file extension to {1}. Please consider removing {2}!'
+
+class NonINIWPTMetaFileHandler(EventHandler):
+    TEST_DIRS_TO_CHECK = (
+        'wpt/metadata',
+        'wpt/mozilla/meta',
+    )
+
+    def on_pr_opened(self, api, payload):
+        diff = api.get_diff()
+        test_dirs_with_offending_files = set()
+
+        for line in diff.split('\n'):
```

Comment:
We should really be ignoring any lines that aren't of the form `diff --git` like the missing_test handler, or we could end up with false positives.